data/questionnaires/brain-development-and-how-children-learn.yml

```diff
@@ -437,7 +425,7 @@ brain-development-and-how-children-learn:
           Complete the statement: The part of the brain that makes sense of the information that comes from our eyes is known as the BLANK
         assessment_summary: 
         assessment_fail_summary: |
-          The occipital lobes are the visual processing area of the brain. You may want to revisit the section on parts of the brain (LINK TO 2.1.1.1).
+          The occipital lobes are the visual processing area of the brain. You may want to revisit the section on parts of the brain.
```

Comment:
this point says 'you MAY want to", the following points say "you MIGHT want to" - do we want to be consistent in the pattern?